ecl/eclcc/eclcc.cpp

```diff
@@ -1976,7 +1976,12 @@ bool EclCC::parseCommandLineOptions(int argc, const char* argv[])
         else if (strcmp(arg, "-internal")==0)
         {
             outputSizeStmts();
-            testHqlInternals();
+            int error = testHqlInternals() + testReservedWords();  //NOTE: testReservedWords() depends on testHqlInternals() so must be be called after.
+            // report test result
+            if (error)
+                printf("%d error%s found!\n", error, error<=1?"":"s");
+            else
+                printf("No errors\n");
```

Comment:
Actually OBT catch the stdout and stderr streams of eclcc when create archive and try to process them to try to determine what happened and generate proper error report if necessary. It would be very helpful if we can assign error codes/return values related to problem or problem group and the test system can use that value and check specific information on the streams to do its job.

Unfortunately I have lack of knowledge the eclcc internals, therefore I can’t give more specific values, but generally I think something like:
- Scanner related problems: 100
- Parser: 200
- Tokenizer: 300
- etc.

It would be helpful. Of course we can refine it later:
e.g: Scanner related problems: 
- 100: general (we have absolutely no idea what is the problem)
- 101: known case1
- 102: known case2 
And so on.